### PR TITLE
Adjusted CCo Loadouts

### PR DESCRIPTION
**Pull request by Robot-Panda15**
**When merged this pull request will:**
- ADDED: Handflares to FTL and up.
- REMOVED: `Microdagrs` from GREN and CLS. 
- CHANGED: Backpacks from `Carryall` to `Assault pack` for Officers. 
- CHANGED: AR scope from `ACOG` to `ELCAN`

### CODE REVIEW

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class CAV_Charlie_OFFCR : CAV_Charlie_Base {  // Leadership positions PSG and up
 
-    backpack[] = {"B_Carryall_mcamo"};
+    backpack[] = {"B_AssaultPack_mcamo"};
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: Can be removed due to being defined in Charlie base as of #267

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp [hunk 2]
```diff
@@ -323,8 +325,6 @@ class CAV_Charlie_CLS : CAV_Charlie_Base {  // Combat Life Saver 24.3kg
         "ACE_MapTools",
         "ACE_CableTie",
 
-        "ACE_microDAGR",
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: Why?

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp [hunk 3]
```diff
@@ -263,9 +268,6 @@ class CAV_Charlie_GR : CAV_Charlie_Base {  // Grenadier 28.4kg
         "ACE_MapTools",
         "ACE_CableTie",
 
-        // Electronics
-        "ACE_microDAGR",
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: Why?

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp [hunk 4]
```diff
@@ -117,7 +120,7 @@ class CAV_Charlie_JFO : CAV_Charlie_Base {  // Joint Fires Observer 36.3kg
 class CAV_Charlie_SL : CAV_Charlie_Base {  // Squad Leader
 
     headgear[] = {"rhsusf_ach_helmet_headset_ess_ocp"};
-    backpack[] = {"B_Carryall_mcamo"};
+    backpack[] = {"B_AssaultPack_mcamo"};
```
Comment (ColdEvul) [inline cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: Can be removed due to being defined in Charlie base as of #267

**Comment by ColdEvul** [comment]
Updated documentation

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp (hunk 2 above)
Comment (Robot-Panda15) [reply cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: Per Company CO

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp (hunk 1 above)
Comment (Robot-Panda15) [reply cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: updated

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp (hunk 4 above)
Comment (Robot-Panda15) [reply cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: updated

File: cScripts/Loadouts/CfgLoadouts_Charlie.hpp (hunk 3 above)
Comment (ColdEvul) [reply cScripts/Loadouts/CfgLoadouts_Charlie.hpp]: rifleman should still have it?

**Pull request merged**